test_conformance/subgroups/subhelpers.h

```diff
@@ -1518,7 +1518,8 @@ template <typename Ty, typename Fns, size_t TSIZE = 0> struct test
 
         // Limit it a bit so we have muliple work groups
         // Ideally this will still be large enough to give us multiple
-        if (local > test_params.local_workgroup_size)
+        if (local > test_params.local_workgroup_size
+            && global % test_params.local_workgroup_size == 0)
```

Comment:
Can you say a little more what this change is doing and why it is required?  Thanks!